README.md

```diff
@@ -133,6 +133,8 @@ regex.
 
 On a new branch, run `npm version <major|minor|patch|...> -m 'rel: %s'` (see
 `npm help version` for details) to update the version in `package.json`, commit
-the changes, and create a git tag. Push the branch, get it reviewed and merged,
-then run `npm publish` from the master branch (you must be logged into the
+the changes, and create a git tag. Push the branch, get it reviewed and merged.
+For the next step, make sure you use npm version 5 or greater (`npm -v`) due to
+our `prepack` script in package.json.
+Run `npm publish` from the master branch (you must be logged into the
```

Comment:
one more break to make it a para? this is kind of in between, won't render as a wrap in markdown.